README.md

```diff
@@ -32,33 +32,24 @@ kernel 4.13**.
 
 **Direct Mode requires a minimum of X.org server 1.20, Linux kernel 4.15 and Mesa 18.2**.
 
-An Ubuntu-packaged version of the Mesa driver can be found in the "Padoka PPA"
-at https://launchpad.net/~paulo-miguel-dias/+archive/ubuntu/pkppa
+An Ubuntu-packaged version of this driver can be found in the "SteamVR Experimental Graphics" PPA
+at https://launchpad.net/~kisak/+archive/ubuntu/steamvr
 
-```
-sudo add-apt-repository ppa:paulo-miguel-dias/pkppa
-sudo apt dist-upgrade
-sudo apt install mesa-vulkan-drivers mesa-vulkan-drivers:i386
-```
-
-Provide your user password when requested and reboot after the last command
-completes to ensure the driver has updated correctly.
-
-Once you've updated Mesa, update your X.org and kernel with SteamVR4Pk PPA
-at: https://launchpad.net/~kisak/+archive/ubuntu/steamvr4pk
+**If using this PPA, make sure a conflicting PPA like oibaf or padoka are not installed.**
 
+To setup "SteamVR Experimental Graphics" run:
 ```
-sudo add-apt-repository ppa:kisak/steamvr4pk
+sudo add-apt-repository ppa:kisak/steamvr
 sudo apt dist-upgrade
-sudo apt install linux-generic-steamvr-18.04
+sudo apt install linux-generic-steamvr-18.04 xserver-xorg-core xserver-xorg-input-evdev xserver-xorg-video-amdgpu mesa-vulkan-drivers mesa-vulkan-drivers:i386
```

Comment:
Whoops, xserver-xorg-input-evdev is a carry over from 16.04. We probably want to recommend xserver-xorg-input-libinput to new users since 18.04 uses it by default. Both should work and is harmless either way.